azure-kusto-data/azure/kusto/data/response.py

```diff
@@ -138,3 +146,88 @@ class KustoResponseDataSetV2(KustoResponseDataSet):
 
     def __init__(self, json_response: List[dict]):
         super(KustoResponseDataSetV2, self).__init__([t for t in json_response if t["FrameType"] == "DataTable"])
+
+
+class KustoStreamingResponseDataSet(BaseKustoResponseDataSet):
```

Comment:
`Progressive`?

Reply:
I removed the progressive wording, see above